### ISSUE TITLE
Accessibility: Keyboard Navigation Skips Submenu

### ISSUE THREAD

**Issue by gioras** [gantry/gantry5#3089]
I am using the core Joomla 4 Skipto Plugin which enable keyboard navigation. However, the keyboard navigation runs through menus and skips submenus.
Thanks
Giora

**Comment by N8Solutions** [comment]
@gioras have you tried using the standard template with Joomla 4, cassiopeia, and see if the same thing happens with your extension?

**Comment by gioras** [comment]
Yes. Cassiopeia works fine.

**Comment by gioras** [comment]
Attached a video of the keyboard navigation skipping the submenus:
https://www.awesomescreenshot.com/video/12157301?key=ff4b898fbb4e08016f02106520e7c1c9

**Comment by gioras** [comment]
@mahagr, @rhukster, @marktaylor46, @simmonsr Any plans to address this bug? this makes Gantry 5 Not accessible according WCAG 2.0.
Your help is very much appreciated.
Giora

**Comment by PhilippSPOTTERON** [comment]
Web Accessibility compliance gets mandatory by law in the EU, by the way - We also have problems with the gantry 5 menu system and using tab on the keyboard.
_"As of 28 June 2025, companies must ensure that the newly marketed products and services covered by the Act are accessible."_
https://ec.europa.eu/social/main.jsp?catId=1202&intPageId=5581&langId=en - this includes all websites by organizations bigger than micro level. 

So the new legislation brings all Gantry 5 users within the EU to the need of real WCAG compliance. Does the Gantry 5 team have already plans for that? Thanks!

**Comment by gioras** [comment]
attached is a fix for this issue

[accessibility_keyboard_fix.zip](https://github.com/user-attachments/files/21582097/accessibility_keyboard_fix.zip)

**Comment by N8Solutions** [comment]
> attached is a fix for this issue
> 
> [accessibility_keyboard_fix.zip](https://github.com/user-attachments/files/21582097/accessibility_keyboard_fix.zip)

Thanks for sharing!
Any chance you can create a pull request for this to make it easier?
